cfgov/v1/tests/wagtail_pages/test_page_titles.py

```diff
@@ -0,0 +1,99 @@
+from django.test import TestCase
+from django.test import Client
+
+from scripts import _atomic_helpers as atomic
+from v1.models.landing_page import LandingPage
+from v1.models.sublanding_page import SublandingPage
+from v1.models.browse_page import BrowsePage
+from v1.models.browse_filterable_page import BrowseFilterablePage, EventArchivePage, NewsroomLandingPage
+from v1.models.sublanding_filterable_page import SublandingFilterablePage, ActivityLogPage
+from v1.models.learn_page import EventPage, LearnPage, DocumentDetailPage
+from v1.models.newsroom_page import NewsroomPage, LegacyNewsroomPage
+from v1.models.blog_page import BlogPage, LegacyBlogPage
+
+
+from helpers import publish_page
+
+
+'''
+Page types tested here: 
+
+LandingPage,
+SubLandingPage,
+BrowsePage,
+BrowseFilterablePage,
+SublandingFilterablePage,
+EventArchivePage,
+DocumentDetailPage,
+LearnPage,
+EventPage,
+NewsroomLandingPage,
+NewsroomPage,
+LegacyNewsroomPage,
+BlogPage,
+LegacyBlogPage,
+ActivityLogPage
+ 
+'''
+
+django_client = Client()
+
+class PageTitlesTestCase(TestCase):
+    """ Tests that all Wagtail page types load and display the title correctly """
+
+    def page_loads_with_correct_title(self, page_type):
+        publish_page(
+            globals()[page_type](
```

Comment:
Ah, I didn't know you could do that!  I'll make that change later; I don't think it's critical.